docker/run.sh

```diff
@@ -79,21 +79,23 @@ done
 
 # Update the arguments to default values if needed.
 
-IMAGE_NAME=${IMAGE_NAME:-ros2_humble_andino}
-CONTAINER_NAME=${CONTAINER_NAME:-ros2_humble_andino_container}
+IMAGE_NAME=${IMAGE_NAME:-ros2_jazzy_andino}
+CONTAINER_NAME=${CONTAINER_NAME:-ros2_jazzy_andino_container}
+
+USER=ubuntu
 
 SSH_PATH=/home/$USER/.ssh
-WORKSPACE_SRC_CONTAINER=/home/$(whoami)/ws/src/$REPOSITORY_FOLDER_NAME
-WORKSPACE_ROOT_CONTAINER=/home/$(whoami)/ws
+WORKSPACE_SRC_CONTAINER=/home/ubuntu/ws/src/$REPOSITORY_FOLDER_NAME
+WORKSPACE_ROOT_CONTAINER=/home/ubuntu/ws
 SSH_AUTH_SOCK_USER=$SSH_AUTH_SOCK
 
 # Create cache folders to store colcon build files
 mkdir -p ${REPOSITORY_FOLDER_PATH}/.build
 mkdir -p ${REPOSITORY_FOLDER_PATH}/.install
 
 # Transfer the ownership to the user
-chown -R "$USER" ${REPOSITORY_FOLDER_PATH}/.build
-chown -R "$USER" ${REPOSITORY_FOLDER_PATH}/.install
+# chown -R "$USER" ${REPOSITORY_FOLDER_PATH}/.build
+# chown -R "$USER" ${REPOSITORY_FOLDER_PATH}/.install
```

Comment:
Is it ok to have it commented?

Reply:
I will remove those lines, I left them JIC I needed to uncomment them on the future.